tracing/opentracing/options.go

```diff
@@ -75,3 +80,10 @@ func WithUnaryRequestHandlerFunc(f UnaryRequestHandlerFunc) Option {
 		o.unaryRequestHandlerFunc = f
 	}
 }
+
+//WithOpName customizes the trace Operation name
```

Comment:
Please add a space after the slash
```suggestion
// WithOpName customizes the trace Operation name
```